flask_homework/views.py

```diff
@@ -0,0 +1,372 @@
+from flask import abort, request, redirect, render_template, session
+from app import app
+from random import choice, randint
+from sessioninfo import *
+from contextdata import *
+
+
+def get_random_name() -> str:
```

Comment:
nit: better to keep views clear, so it is a good idea to move data somewhere in fixtures or instance module

Reply:
Agree. I also was in doubt if any accessory functions should be separated to other file to avoid some mess. Maybe it's better to spend a minute or two on the lesson explaining trainees how the project is preferred to be structured.